runtime/jvmti/jvmtiHeap.c

```diff
@@ -1137,7 +1137,20 @@ mapEventType(J9JVMTIHeapData * data, IDATA type, jint index, j9object_t referrer
             event->type = J9JVMTI_HEAP_EVENT_OBJECT;
```

Comment:
While you're here, please fix this line to use tabs to indent.

Reply:
Done.